deepmd/pt/model/atomic_model/base_atomic_model.py

```diff
@@ -363,6 +363,7 @@ def compute_or_load_stat(
         self,
         merged: Union[Callable[[], list[dict]], list[dict]],
         stat_file_path: Optional[DPPath] = None,
+        compute_out_stat: bool = True,
```

Comment:
Please note that the docs for this method needs to be changed. (Other  methods have the same issue)